### ISSUE TITLE
Possibility to recalculate `op_str` when changing for example the displacement 

### ISSUE THREAD

**Issue by koemeet** [aquynh/capstone#921]
It would be nice if it was possible to alter the `disp` member of the a `cs_x86` object and calculate a new operands string for that instruction. Is it maybe already possible to rebuild the operands string, given some `cs_insn` or `cs_x86` object?

**Comment by aquynh** [comment]
Yes, i am going to add a new API for this, then you can feed the newly generated code to Keystone, for example.

**Comment by koemeet** [comment]
@aquynh Awesome!
